src/util.cc

```diff
@@ -3459,20 +3452,18 @@ do_copy (const char *source, GnomeVFSURI *source_uri,
 
   start_entertaining_user (TRUE);
 
-  result = gnome_vfs_async_xfer (&handle,
-				 source_uri_list,
-				 target_uri_list,
-				 GNOME_VFS_XFER_DEFAULT,
-				 GNOME_VFS_XFER_ERROR_MODE_QUERY,
-				 GNOME_VFS_XFER_OVERWRITE_MODE_REPLACE,
-				 GNOME_VFS_PRIORITY_DEFAULT,
-				 copy_progress,
-				 NULL,
-				 NULL,
-				 NULL);
+  g_file_copy_async (source_uri, // GFile *source
+                     target_uri, // GFile *destination
+                     G_FILE_COPY_TARGET_DEFAULT_PERMS, // GFileCopyFlags flags
+                     G_PRIORITY_DEFAULT, // int io_priority
+                     cancellable, // GCancellable *cancellable
+                     copy_progress, // GFileProgressCallback progress_callback
+                     cancellable, // gpointer progress_callback_data
+                     NULL, // GAsyncReadyCallback callback
+                     NULL); // gpointer user_data
```

Comment:
What are these "//" comments for?

Reply:
Removed. Leftover when I needed them for reference.